include/rocksdb/utilities/backupable_db.h

```diff
@@ -165,6 +165,33 @@ struct BackupableDBOptions {
   // db_session_id as a fallback.
   BackupTableNameOption share_files_with_checksum_naming;
 
+  // Option for custom checksum functions.
+  // When this option is nullptr, BackupEngine will use its default crc32c as
+  // the checksum function.
+  //
+  // When it is not nullptr, this option comes into effect only if DB has a
+  // custom checksum factory and this option is set to the same factory.
+  //
+  // Regardless of what is set for this option, BackupEngine will always use
+  // its default crc32c checksum for creating, verifying, or restoring backups.
+  //
+  // If this option is set to the DB checksum factory, BackupEngine will try
+  // to find the requested checksum function and use it to create, verify, or
```

Comment:
Better to explain what is the "requested checksum function" here. It is unclear what's the relationship between this option, DB checksum factory and requested checksum function.